tests/src/Traits/OgTrait.php

```diff
@@ -70,4 +72,33 @@ protected function convertOgRoleNamesToIds(array $roles, EntityInterface $entity
     return $roles;
   }
 
+  /**
+   * Asserts the ownership of a user to a given group.
+   *
+   * An ownership is defined as having a specific collection of roles in that
+   * group.
+   *
+   * @param AccountInterface $user
+   *    The user to be checked.
+   * @param RdfInterface $group
+   *    The group entity. In this project, only rdf entities are groups.
```

Comment:
Well we are using this only for rdf entities and it is project specific. So I don't think it should change here. because we are also securing the proper entity type passed in.